splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkTrustManager.java

```diff
@@ -0,0 +1,22 @@
+package com.redhat.console.notifications.splunkintegration;
+
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import javax.net.ssl.X509TrustManager;
+
+class SplunkTrustManager implements X509TrustManager {
```

Comment:
Similar to method naming for the ssl context, can this also have name that represents that it trusts anything?

Reply:
Changing it to `SplunkTrustAllCACerts.java`